api/services/v1alpha1/auth_types_test.go

```diff
@@ -5,66 +5,8 @@ import (
 
 	. "github.com/onsi/gomega"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
```

Comment:
Removing these test cases is intentional? is gateway implementation is in initial stage, so removing these auth testcases are very early.

Reply:
ok api/services/v1alpha1/types_test.go is become generic

Reply:
I don't understand this. @resoluteCoder, why are you removing a unit test?